features/buyer/cancel_requirements.feature

```diff
@@ -0,0 +1,60 @@
+@cancel
+Feature: Cancel a requirement
+  In order to ensure the procurement process is fair and transparent
+  As a buyer within government
+  I want to be able to publish the result of my procurement process
```

Comment:
I don't see how this feature description relates to the heading!

Reply:
Cancelling the requirement would be a result of my procurement process, just as awarding is one, right?

Reply:
Sort-of, it means there isn't a procurement process. But shouldn't the "Feature" at the top of a file reflect exactly what's in the file (no more no less), in which case it should mention cancelling specifically as the user story in question? Maybe I misunderstood how these are supposed to look..

Reply:
ok, I found a more appropriate one I think: "As a buyer, I need to cancel my requirement after applications have closed, because I didn't end up going ahead with the procurement."

Is it ok? :)

Reply:
Now updated

Reply:
👍